runtime/runtime/src/actions.rs

```diff
@@ -458,12 +474,16 @@ pub(crate) fn action_implicit_account_creation_transfer(
     account: &mut Option<Account>,
     actor_id: &mut AccountId,
     account_id: &AccountId,
-    transfer: &TransferAction,
+    deposit: Balance,
     block_height: BlockHeight,
     current_protocol_version: ProtocolVersion,
+    nonrefundable: bool,
```

Comment:
Will it be possible to create an implicit account with both a normal balance an a non-refundable balance? I think it should be possible, and in that case there should be two separate `Balance` arguments instead of `Balance` and `bool`.

Reply:
It is possible now, but for implicit account creation `NonrefundableStorageTransfer` must be the first action in the receipt. I could allow for regular transfer first, but this would require more invasive modification to the actions processing logic and we can always introduce this later.

Reply:
Sure, requiring a specific order is ok I think. We just need to make sure it's documented somewhere.